apps/docs/src/app/global.css

```
@@ -1,106 +1,121 @@
+   original CSS @font-face names in case --font-mona-sans / --font-mona-mono
+   are ever absent. */
+:root {
+    --font-sans-display:
+        var(--font-mona-sans, "Mona Sans VF"), "Inter", "Roboto",
+        "Helvetica Neue", "Arial Nova", "Nimbus Sans", "Arial", sans-serif;
+    --font-mono:
+        var(--font-mona-mono, "Mona Sans Mono VF"), ui-monospace,
+        "Cascadia Code", "Source Code Pro", "Menlo", "Consolas",
+        "DejaVu Sans Mono", monospace;
+}
 @import "fumadocs-ui/css/shadcn.css";
```

Stylelint (17.9.0)

[error] Invalid position for `@import` rule (no-invalid-position-at-import-rule)

(no-invalid-position-at-import-rule)